api/v3/openapi.yaml

```diff
@@ -627,6 +627,45 @@ paths:
         '200':
           $ref: '#/components/responses/GetReplicaResponse'
 
+  /clusters/{cluster_id}/topics/{topic_name}/records:
+    parameters:
+      - $ref: '#/components/parameters/ClusterId'
+      - $ref: '#/components/parameters/TopicName'
+      - $ref: '#/components/parameters/PartitionId'
+
+    post:
+      summary: 'Produce records to the given topic.'
+      description: '<p>Produce records to the given topic, returning delivery reports for each
+                    record produced.</p>
+                    <p>This is a streaming API. To use it in streaming mode, you need to specify the
+                    "Transfer-Encoding: chunked" header. For as long as the connection is kept open,
+                    the server will keep accepting records. For each record sent to the server, the
+                    server will asynchronously send back a delivery report, in the same order.
+                    Records are streamed to and from the server as Concatenated JSON.</p>
+                    <p>Errors are reported per record. The HTTP status code will be HTTP 200 OK as
+                    long the connection is successfully established.</p>'
+      tags:
+        - Records
+      requestBody:
+        $ref: '#/components/requestBodies/ProduceRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/ProduceResponse'
+
+
+  /clusters/{cluster_id}/topics/-/configs:
+    parameters:
+      - $ref: '#/components/parameters/ClusterId'
+
+    get:
+      summary: 'Get All Topic Configs'
+      description: 'Returns all topic configurations for topics hosted by the specified cluster.'
+      tags:
+        - Configs
+      responses:
+        '200':
+          $ref: '#/components/responses/ListTopicConfigsResponse'
```

Comment:
nit: This is just fixing an unrelated omission, right?

Reply:
This was deleted on another PR, and came back when merging. I missed it, sorry.